bots/psuteam7bot/movement.js

```diff
@@ -158,24 +165,35 @@ movement.checkQuadrant = (location, fullmap) => {
     }
 }
 
-/*Calculate and return enemy castle's potential starting location
+/**
+*Calculate and return enemy castle's potential starting location
 *Input:     myCastleLocation    -   the Castle's 'position/ location' object, should be self.me
 *           fullMap             -   the full map, Should be self.map or or self.getPassableMap()
-*Output:    RetVal  -   An array containing 3 {x, y} objects, 2 of which are the potential 'location' of enemy Castle mirrorring base, 
-                        and the third a position of an enemy quadrant to check if the mirror castle is destroyed
+*Output:    RetVal  -   An array containing 2 {x, y} objects, One is the enemy castle mirroring myCastleLocation, 
+                        and the second a position of an enemy quadrant to check if the mirror castle is destroyed
 *
 *TODO: Have the team's castles communicate at start of game to potentially improve accuracy. 
 *Given the team's starting castles are in different quadrants, can accurately calculate location of enemy castle
```

Comment:
I agree that this is a long-term goal, but I like what you have currently as a general goal. If we are seeing lots of issues we can tweak.